python/web/list.py

```diff
@@ -0,0 +1,110 @@
+
+import cgi
+import glob
+from html.parser import HTMLParser
+
+from .index import capture_output
```

Comment:
This file seems redundant now with the create_list() method in `index.py`.  Is it used or vestigial?